presto-product-tests/conf/presto/etc/singlenode-kerberized.properties

```diff
@@ -10,7 +10,6 @@ node.environment=test
 
 coordinator=true
 node-scheduler.include-coordinator=true
-http-server.http.port=8080
```

Comment:
1. This is default port anyway. You need also `http-server.http.enabled = false`

2. Commit message "Hide unsecure coordinator server port" -- "hide" suggests hiding open port. What about "Do not open insecure ... "
(also `s/unsecure/insecure`)

Reply:
Thanks. BTW: https://english.stackexchange.com/a/19659  👍